spec/controllers/admin/ip_based_groups_controller_spec.rb

```diff
@@ -0,0 +1,217 @@
+require 'rails_helper'
+
+RSpec.describe Admin::IpBasedGroupsController, :type => :controller do
+  let(:roles) { [instance_double(Role)] }
+  let(:groups_factory) { class_double(IpBasedGroup) }
+  before do
+    sign_in FactoryGirl.create(:user, :admin)
```

Comment:
So I've seen ``` sign_in(user) if user ``` and ```sign_in user ``` and now this. I think we should try to stick to one of the above.